Fly_Swatter/controls.py

```diff
@@ -0,0 +1,50 @@
+from Fly_Swatter.Fly_Swatter import radar
```

Comment:
controls file is a continued version of the mode_control file and one that is much easier used by the gui and probably CLI